### PR TITLE
server+tapchannel: modify PaymentBandwidth to return link bandwidth

### PR DESCRIPTION
**Pull request by ffranr**
Dependant on https://github.com/lightningnetwork/lnd/pull/8852
Requires LND version bump once the above PR is merged. Will remove from draft when ready for merge.

This PR is ready for review now.

In this PR, the traffic shaper's `PaymentBandwidth` method is modified to return the link bandwidth if the tap channel commitment blob or HTLC record is absent.

This modification ensures the traffic shaper returns the correct bandwidth for tap channels used as standard BTC lightning channels, ignoring any committed taproot assets.

### CODE REVIEW

File: tapchannel/aux_traffic_shaper.go [hunk 1]
```diff
@@ -120,13 +120,15 @@ func (s *AuxTrafficShaper) HandleTraffic(_ lnwire.ShortChannelID,
...
-	// This method shouldn't be called if we don't have a commitment blob
-	// available (i.e., the channel is not a custom channel).
+	// If the commitment or HTLC blob is not set, we don't have any
+	// information about the channel and cannot determine the available
+	// bandwidth from a taproot asset perspective. We return the link
+	// bandwidth as a fallback.
 	if commitmentBlob.IsNone() || htlcBlob.IsNone() {
-		return 0, fmt.Errorf("no commitment or TLV blob available " +
-			"for custom channel bandwidth estimation")
+		return linkBandwidth, nil
```
Comment (Roasbeef) [inline tapchannel/aux_traffic_shaper.go:131]: 👍

**Comment by guggero** [comment]
Pushed up some fixes, verified through https://github.com/lightninglabs/lightning-terminal/pull/775.

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Review by GeorgeTsagk — APPROVED** [review]
Looks good ✅

**Pull request merged**